run_elixir

```diff
@@ -2,10 +2,9 @@
 
 export TIMEFORMAT=%R
 cd elixir/
-mix compile.protocols
```

Comment:
Good call. :+1: for leaving it then! No need to explicitly add the path
anymore though.

-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Lead Developer*